apps/test-blinky/src/tutorial_servo_control_can.cpp

```diff
@@ -0,0 +1,22 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"
+#include "TutorialServo.h"
+// Blinking rate in milliseconds
+#define BLINKING_RATE 500ms
+AnalogIn potVoltageIn(PA_0);
+PwmOut servoPwmOut(PA_1);
+servo1 TutorialServo(PA_1, 180.0, 1, 2);
```

Comment:
i think u swapped the order here. u have name type() when it should be type name()